### ISSUE TITLE
Logistic Regression probability scores wrong in case of string labels and multi_class set to multinomial.

### ISSUE THREAD

**Issue by prabhat00155** [onnx/sklearn-onnx#147]
data = load_digits()
X = data.data
y = data.target

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)
model = LogisticRegression(random_state=42, multi_class='multinomial', solver='sag').fit(X_train, y_train)

onnx_model = convert_sklearn(model, 'lr', [('input', FloatTensorType(X_test.shape))])
save_model(onnx_model, 'lr.onnx')
sess = InferenceSession('lr.onnx')
res = sess.run(None, input_feed={'input': X_test.astype(np.float32)})

np.mean(np.isclose(model.predict_proba(X_test), list(map(lambda x:
                                                         list(map(lambda y: x[y], x)), res[1]))))

>> 0.010901001112347052
